packages/client-core/src/user/components/UserMenu/menus/ShareMenu.tsx

```diff
@@ -91,6 +92,14 @@ const ShareMenu = (): JSX.Element => {
     refLink
   })
 
+  useEffect(() => {
```

Comment:
since this service is used by various systems, we should be registering this in `public.tsx` along with the other receptors there